LICENSE.txt

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2012-18 Bozhidar Batsov
```

Comment:
@bbatsov Development of RuboCop Rails will start in 2018, so perhaps it may be better to change to 2018 start. Now I copied RuboCop Core and it is 2012 start. What do you think about it?

Reply:
I also left 2012 in my RuboCop Performance PR. I will follow whatever we decide here.

Reply:
I guess it's fine either way - after all much of the code was written before the new projects were created.

Reply:
I got it. Thanks for your reply.